robbot.py

```diff
@@ -26,6 +16,33 @@
 
 bot = commands.Bot(command_prefix='.')
 
+'''
+    Cog Functions & Commands
+'''
+def load_cogs():
+    for files in os.listdir('./cogs'):
+        if files.endswith('.py'):              
+            bot.load_extension(f'cogs.{files[:-3]}')
+            print(f'{files[:-3]} extension loaded')
+
+def unload_cogs():
+    for files in os.listdir('./cogs'):
+        if files.endswith('.py'):              
+            bot.unload_extension(f'cogs.{files[:-3]}')
+            print(f'{files[:-3]} extension unloaded')
+
+@bot.command(name='refresh', description='Reloads all cogs', aliases=['reload'], hidden=True)
+async def refresh(ctx: commands.Context):
+    print('Reloading Cogs...\n')
+    unload_cogs()
+    load_cogs()
+    print('Cogs reloaded successfully')
+    reaction = '👍'
+    await ctx.message.add_reaction(reaction)
```

Comment:
I'm not sure about this yet, but I think this might be antithetical to the purpose of cogs & extensions. If I understand them right, the idea is to allow reloading of individual cogs at runtime without affecting the rest of the cogs, e.g. `bot.reload_extension('Music')` rather than unloading and loading all of them.

See [this example](https://github.com/Rapptz/RoboDanny/blob/master/cogs/admin.py#L18) I found that I think demonstrates this point better than I can explain. Also note that he has the commands in an `Admin` cog, probably a good idea for us as well.